euth/ideas/models.py

```diff
@@ -20,6 +22,11 @@ def get_absolute_url(self):
         from django.core.urlresolvers import reverse
         return reverse('idea-detail', args=[str(self.slug)])
 
+    def clean(self):
+        super().clean()
```

Comment:
If you [take a look at the source code](https://docs.djangoproject.com/en/1.8/_modules/django/db/models/base/#Model.clean) you will recognize that `clean()` is just an empty hook. So no need to call `super()` here. 😄